sql/upgrade/1.4.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -168,6 +268,199 @@ delimiter ;
 CALL ct_dolphin_T_t_ds_process_definition_version;
 DROP PROCEDURE ct_dolphin_T_t_ds_process_definition_version;
 
+-- uc_dolphin_T_t_ds_project_instance_A_add_code
+drop PROCEDURE if EXISTS uc_dolphin_T_t_ds_project_instance_A_add_code;
+delimiter d//
+CREATE PROCEDURE uc_dolphin_T_t_ds_project_instance_A_add_code()
+   BEGIN
+       IF NOT EXISTS (SELECT 1 FROM information_schema.COLUMNS
+           WHERE TABLE_NAME='t_ds_project'
+           AND TABLE_SCHEMA=(SELECT DATABASE())
+           AND COLUMN_NAME ='code')
+   THEN
+         ALTER TABLE t_ds_project ADD `code` bigint(20) NOT NULL COMMENT 'encoding';
+       END IF;
+ END;
+
+d//
+
+delimiter ;
+CALL uc_dolphin_T_t_ds_project_instance_A_add_code();
+DROP PROCEDURE uc_dolphin_T_t_ds_project_instance_A_add_code;
+
+-- uc_dolphin_T_t_ds_process_definition_A_add_code
+drop PROCEDURE if EXISTS uc_dolphin_T_t_ds_process_definition_A_add_code;
```

Comment:
Please update UNIQUE KEY of t_ds_process_definition

Reply:
thanks @JinyLeeChina , solved it.